forem/forem
# Update the UI for Low-Quality-Comments Notice

## What type of PR is this? (check all applicable)


## Description
Updates the notice that appears above comments that have been flagged as `low-quality` by the forem's community. The updated UI is designed to clearly communicate the the forem's admins do not support such low-quality content, or allow its propagation through the forem. See [this diagram](https://user-images.githubusercontent.com/816402/108918129-b9cb4880-75e5-11eb-84aa-2688013496c6.jpg) for more context.

## Related Tickets & Documents
closes https://github.com/forem/forem/issues/12751

## QA Instructions, Screenshots, Recordings
1. Login to `localhost` as a `trusted` user. Navigate to an article's comment. Click on the three-dot icon in the top-right corner of the comment. Click on `Moderate`
2. Choose the `Flag To Admins` option in order to downgrade the comment's quality rating past the `-100` mark, to trigger the low-quality notice.
3. Go back to the article page and refresh. You should observe the following:
- The comment has moved lower in the comment list.
- There is a yellow `crayons-warning` notice with an explanation and a link to the Code of Conduct.
- The comment text has been slightly greyed out.

Please see a Loom video below demonstrating these steps 🙏🏾 
https://www.loom.com/share/1251ede2f8904939961d0cc712587072

### UI accessibility concerns?

This PR does not introduce any accessibility concerns; the notice's design leverages our Crayons design system, which has already been properly vetted.

## Added tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![South Park GIF](https://media.giphy.com/media/3o6Zt9CYYAHKeHqnqU/giphy.gif)
